### PR TITLE
new icon: renpy (original, plain) 

### LINKED ISSUE #1179: [ICON REQUEST]: Ren'Py
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

Ren'Py is the most popular visual novel engine

### Links and sources

- Logo: https://www.renpy.org/doc/html/_static/navbar-logo.png
- Github: https://github.com/renpy/renpy
- Website: https://www.renpy.org/

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by lunatic-fox**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #1179**

## Link to prove your SVG is correct and up-to-date.
https://github.com/renpy/renpy/blob/master/launcher/game/images/logo.png

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -6292,6 +6292,27 @@
...
         ]
     },
+    {
+        "name": "renpy",
+        "altnames": [
+            "Ren'Py"
+        ],
+        "tags": [
+            "programming",
+            "language"
```
Comment (Panquesito7) [inline devicon.json:6302]: I'm not sure this fits for a programming language but more for a game engine.
```suggestion
            "programming",
            "game-engine",
            "engine",
            "python"
```

Comment (lunatic-fox) [reply devicon.json:6302]: I'm not a user of `Ren'Py` nor `Python`, but I agree with you.
`Ren'Py` calls itself as a visual novel engine and looking through docs it's said:
> "Many portions of Ren'Py take Python expressions."

> "While Python expressions are very powerful, only a fraction of that power is necessary to write a basic Ren'Py game."

Looking through some `.rpy` files they look as a flavored `Python` to me, agreeing to the game engine idea.
That was my first contribution and now I got the general idea of those tags. Thank you!

**Review by Panquesito7 — APPROVED** [review]
Looks great. Thanks for contributing! 🚀

**Comment by Panquesito7** [comment]
CC: @BenSouchet.
Would you like to review this and other PRs? Thank you. 😄

**Review by Snailedlt — APPROVED** [review]
Looks great!
Thank you for your contribution! :heavy_check_mark:

**Comment by BenSouchet** [comment]
@Panquesito7 Hi, I would be happy to help by reviewing PRs , and / or help sorting new issues when I have time :)

**Pull request merged**